packages/grafana-data/src/types/select.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Used in select elements
+ */
+export interface SelectOptionItem<T = any> {
```

Comment:
it does feel strange to have it in data, but think it's ok. Maybe makes more sense if we change the name, the name feels a bit too UI and the doubling of Option & Item also feels a bit strange. 

SelectableItem ? 

Also remember that typescript has duck typing, you do not need to use the same type for things, just as long as it as a matching signature/props. So another alternative to moving this Type to data is to have two similar interfaces  (one defined here), and one defined in grafana/ui.

Reply:
I like SelectableItem 👍 

one interface seems better than two -- but yes, that is another reasonable option

Reply:
ok, lets keep it in @grafana/data in that case but rename it to SelectableItem